src/new-banlist.ts

```diff
@@ -0,0 +1,174 @@
+// Create a new banlist based on an existing banlist and additions/deletions.
+//
+// Currently only handles standard implicitly.
+// Example usage:
+//   npm run build
+//   npx ts-node dist/new-banlist.js -s standard-ban-list-23-09 -n "Standard Banlist 24.03" --effective_date="2024-03-18" -c standard-ban-list-24-03 -a formicary
+
+import fs from "fs";
+import { resolve } from "path";
+import commandLineArgs = require('command-line-args');
+import prettier from "prettier";
+import { getMwlJson, getRestrictionsV2Json, textToId, getPrintingsV2Json } from "./index";
+
+async function formatJSON(text: any) {
+  const data = JSON.stringify(text);
+  return await prettier.format(data, {
```

Comment:
You don't need to await here, as you're returning the promise either way.

Reply:
ah, thanks.